mapdata/WarcraftLegacies/AbilityData/Skin/1163210817.json

```diff
@@ -23,23 +23,18 @@
     },
     {
       "Id": 2020631137,
-      "Value": 2
+      "Value": 3
     },
     {
       "Level": 1,
       "Id": 829453409,
       "Type": 3,
-      "Value": "Necromantic Cleansing"
+      "Value": "Dispel"
```

Comment:
Should be called "Dispel Magic" to match Priest version. Good change it from Necromantic Cleansing though.